src/ripple/nodestore/impl/DatabaseRotatingImp.h

```diff
@@ -34,21 +34,13 @@ class DatabaseRotatingImp : public DatabaseRotating
     operator=(DatabaseRotatingImp const&) = delete;
 
     DatabaseRotatingImp(
-        std::string const& name,
         Scheduler& scheduler,
         int readThreads,
-        Stoppable& parent,
         std::shared_ptr<Backend> writableBackend,
         std::shared_ptr<Backend> archiveBackend,
         Section const& config,
         beast::Journal j);
 
-    ~DatabaseRotatingImp() override
-    {
-        // Stop read threads in base before data members are destroyed
-        stopReadThreads();
```

Comment:
@scottschurr  I believe with the new top-down approach, since the Application object owns the nodestore, it must call the nodestore stop() after its stop() is called.